paasta_tools/cli/cmds/validate.py

```diff
@@ -580,6 +583,22 @@ def validate_autoscaling_configs(service_path):
                 is False
             ):
                 autoscaling_params = instance_config.get_autoscaling_params()
+                if autoscaling_params["metrics_provider"] in {
+                    "active-requests",
+                }:
```

Comment:
this is fine as-is, but technically
```suggestion
                if autoscaling_params["metrics_provider"] == "active-requests":
```
would be a little more accurate if we're thinking that there won't be another metrics provider with these same params :)